lib/client/backend_client.dart

```diff
@@ -1,23 +1,51 @@
+//import 'dart:convert';
+
+import 'dart:convert';
+
 import 'package:http/http.dart' as http;
 
 class BackendClient {
   const BackendClient();
 
 //functions to connect with API
-
-  Future<dynamic> getData({required String uri}) async {
+  Future<List<Map<String, String>>> getData({required String uri}) async {
     var url = Uri.http('10.0.2.2:8080', uri);
```

Comment:
Please save this value as a constant. It could be imported from a constant file or be a `static const` variable at the top of the file.

In the end, we might deploy this backend somewhere! It would be great to be one step closer to being configurable because if we are hitting a back-end in AWS or something, it won't be at http://10.2.2.2/.

In general, literal values are discouraged. Saving them as a variable gives the literal value a name (the variable name) and it adds context as you read the code.

Reply:
done!